node/sc/bridge_manager.go

```diff
@@ -335,23 +346,29 @@ func (bi *BridgeInfo) handleRequestValueTransferEvent(ev *RequestValueTransferEv
 	return nil
 }
 
-// UpdateRequestNonceFromCounterpart updates the request nonce from counterpart bridge.
-func (bi *BridgeInfo) UpdateRequestNonceFromCounterpart(nonce uint64) {
+// SetRequestNonceFromCounterpart sets the request nonce from counterpart bridge.
+func (bi *BridgeInfo) SetRequestNonceFromCounterpart(nonce uint64) {
 	if bi.requestNonceFromCounterPart < nonce {
 		vtRequestNonceCount.Inc(int64(nonce - bi.requestNonceFromCounterPart))
 		bi.requestNonceFromCounterPart = nonce
 	}
 }
 
-// UpdateRequestNonce updates the request nonce of the bridge.
-func (bi *BridgeInfo) UpdateRequestNonce(nonce uint64) {
+// SetRequestNonce sets the request nonce of the bridge.
+func (bi *BridgeInfo) SetRequestNonce(nonce uint64) {
 	if bi.requestNonce < nonce {
 		bi.requestNonce = nonce
 	}
 }
 
-// UpdateHandledNonce updates the handled nonce with new nonce.
-func (bi *BridgeInfo) UpdateHandledNonce(nonce uint64) {
+// MarkHandledNonce marks the handled nonce and set the handle nonce value.
+func (bi *BridgeInfo) MarkHandledNonce(nonce uint64) {
+	bi.SetHandleNonce(nonce)
+	bi.handledEvent.PutWithLimit(requestEvent{nonce}, 10000000)
```

Comment:
Is there a problem that it may be added to handle nonce but not put in event because of limit?

Reply:
No, there is no problem. 
If a transaction tries to handle the nonce, it will be prevented in the bridge contract.